qchem/pennylane_qchem/__init__.py

```diff
@@ -14,5 +14,5 @@
 """The PennyLane quantum chemistry package. Supports OpenFermion, PySCF,
 and Psi4 for quantum chemistry calculations using PennyLane.
 """
-from . import qchem
+from .qchem import hf
```

Comment:
are there any consequences from removing the `from . import qchem` line?

Reply:
It is already changed to `from .qchem import hf`, not sure why it showed the older version.